### PR TITLE
is.empty(null) should return true

### LINKED ISSUE #13: is.empty(null) returns false (should return true?)
I just noticed that `is.empty(null)` does not return `false` like I would expect. I'm thinking it should return `true`, is this a bug or an intentional decision?

### PR DESCRIPTION
**Pull request by dominicbarnes**
Fixes #13

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -88,6 +88,10 @@ is.empty = function (value) {
     return true;
   }
 
+  if (is.nil(value)) {
```
Comment (ljharb) [inline index.js]: I wonder if this would be simpler as `return !!value` which covers `null`, `undefined`, and `NaN`?

Comment (dominicbarnes) [inline index.js]: I'd be down for that, just wasn't sure how far-reaching my change should be. I'll make that change right now.

Comment (dominicbarnes) [inline index.js]: Fixed with 9beded94c6a856e234f2eb8d76d816f2c69552cd

**Comment by ljharb** [comment]
Thanks!

**Pull request merged**